swagger/sales/coupon.json

```diff
@@ -59,6 +59,15 @@
                   "description": "Filter parameter coupon status, optional values = [active / expiered / scheduled / still_available]",
                   "required": false
                 },
+                {
+                  "name": "uid_list",
+                  "in": "query",
+                  "schema": {  
+                    "type": "string" 
+                  },
+                  "description": "Filter parameter string of Coupons uid separated by comma",
+                  "required": false
```

Comment:
Change description to:
Filter the coupon list based on a list of coupon UIDs supplied in a comma-separated list.

Eldad told me that you can feed GET with an Array instead of comma-separated list. Please ask him about it

Reply:
It is an array.
I've change he description accordingly.